duckduckgo.pot

```diff
@@ -1878,6 +1878,12 @@ msgctxt "safe search"
 msgid "Don't filter adult content"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. Headline on the Mountain Climber Onboarding slide
+msgctxt "onboarding_slide"
+msgid "Don't let friends get tracked!"
+msgstr ""
```

Comment:
@brianhall, it seems like this string (as well as a few more below) that want to ride along on this PR. Any concern with sending these to Smartling?

Reply:
No concerns at all, has been on my to do list. Thank you, @chuckharmston!